build/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
2,207],[173,182,202],[71,72,73,74,202],[65,76,77,78,79,202],[82,202],[65,71,82,84,85,86,87,88,91,202],[65,82,85,89,90,91,93,94,95,96,97,98,202],[89,90,202],[84,202],[63,202],[64,66,67,68,70,75,80,83,92,99,202],[69,202],[71,72,74,75,202],[77,79,202],[65,71,82,86,87,88,202],[64,66,67,68,114,115,116,117,202],[65,71,72,73,120,121,202],[65,76,77,79,120,202],[65,71,82,85,86,120,127,128,202],[65,82,85,120,124,125,126,130,131,132,133,134,135,202],[65,124,125,202],[126,202],[137,202],[66,120,121,122,123,129,136,138,139,140,202],[65,67,126,127,137,202],[65,81,202],[65,202],[63,65,66,202],[65,71,202],[68,202]],"referencedMap":[[101,1],[103,2],[104,2],[102,2],[105,2],[106,2],[107,1],[108,1],[109,2],[110,2],[111,2],[112,2],[113,2],[119,3],[142,4],[144,4],[145,4],[143,4],[146,4],[147,4],[148,5],[149,4],[150,4],[151,4],[152,4],[153,4],[154,4],[81,6],[156,7],[157,7],[159,8],[160,9],[161,10],[162,11],[163,12],[164,13],[165,14],[166,15],[167,16],[168,17],[169,17],[171,18],[170,19],[172,18],[173,20],[174,21],[158,22],[208,6],[175,23],[176,24],[177,25],[209,26],[178,27],[179,28],[180,29],[181,30],[182,31],[183,32],[184,33],[185,34],[186,35],[187,36],[188,36],[189,37],[190,38],[192,39],[191,40],[193,41],[194,42],[195,6],[196,43],[197,44],[198,45],[199,46],[200,47],[201,48],[202,49],[203,50],[204,51],[205,52],[206,53],[207,54],[61,6],[62,6],[12,6],[11,6],[2,6],[13,6],[14,6],[15,6],[16,6],[17,6],[18,6],[19,6],[20,6],[3,6],[21,6],[4,6],[22,6],[26,6],[23,6],[24,6],[25,6],[27,6],[28,6],[29,6],[5,6],[30,6],[31,6],[32,6],[33,6],[6,6],[37,6],[34,6],[35,6],[36,6],[38,6],[7,6],[39,6],[44,6],[45,6],[40,6],[41,6],[42,6],[43,6],[8,6],[49,6],[46,6],[47,6],[48,6],[50,6],[9,6],[51,6],[52,6],[53,6],[56,6],[54,6],[55,6],[57,6],[58,6],[10,6],[59,6],[1,6],[60,6],[155,55],[75,56],[80,57],[83,58],[92,59],[99,60],[91,61],[78,6],[93,6],[94,6],[63,6],[69,6],[95,6],[96,6],[87,62],[97,6],[88,6],[98,6],[89,6],[90,6],[64,63],[100,64],[70,65],[114,66],[115,67],[116,68],[117,58],[118,69],[122,70],[123,71],[129,72],[136,73],[126,74],[121,6],[120,6],[130,6],[131,6],[137,6],[132,6],[133,6],[127,75],[134,6],[128,6],[135,6],[124,6],[125,6],[138,76],[141,77],[139,78],[140,6],[71,6],[82,79],[76,6],[84,80],[72,80],[77,6],[79,6],[85,6],[67,81],[73,82],[86,6],[66,80],[74,83],[68,6],[65,6]],"affectedFilesPendingEmit":[101,103,104,102,105,106,107,108,109,110,111,112,113,119,142,144,145,143,146,147,148,149,150,151,152,153,154,155,75,80,83,92,99,91,78,93,94,63,69,95,96,87,97,88,98,89,90,64,100,70,114,115,116,117,118,122,123,129,136,126,121,120,130,131,137,132,133,127,134,128,135,124,125,138,141,139,140,71,82,76,84,72,77,79,85,67,73,86,66,74,68,65],"emitSignatures":[63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155]},"version":"5.5.4"}
```

Comment:
nit: can we add `build` to .gitignore in sync with change proposed in #290 please - hasn't been commited to `main` yet and would be nice to avoid !